src/main/ts/components/Editor.tsx

```diff
@@ -46,25 +48,25 @@ export class Editor extends React.Component<IAllProps> {
   };
 
   private id: string;
-  private elementRef: React.RefObject<Element>;
-  private editor?: Record<any, any>;
+  private elementRef: React.RefObject<HTMLElement>;
+  private editor?: TinyMCEEditor;
   private inline: boolean;
-  private currentContent?: string | null;
-  private boundHandlers: Record<string, EventHandler<any>>;
+  private currentContent?: string;
+  private boundHandlers: Record<string, (event: EditorEvent<unknown>) => unknown>;
 
   constructor(props: Partial<IAllProps>) {
     super(props);
     this.id = this.props.id || uuid('tiny-react');
-    this.elementRef = React.createRef<Element>();
-    this.inline = this.props.inline ? this.props.inline : this.props.init && this.props.init.inline;
+    this.elementRef = React.createRef<HTMLElement>();
+    this.inline = this.props.inline !== undefined ? (this.props.inline ?? false) : ((this.props.init && this.props.init.inline) ?? false);
```

Comment:
Can this be simplified?

Maybe something like:
```
!!(this.props.inline || this.props?.init?.inline)
```

Reply:
Hmmmm... Well I suppose though the two are technically different.

For example:
```jsx
<Editor inline={false} init={{inline: true}} />
```
Would result in Iframe mode in the current code but inline mode in your code.

I was thinking about the problem as the inline prop should override the init prop, whereas your version essentially just 'ORs' them together.

Reply:
Maybe this:
```javascript
this.props.inline ?? this.props?.init?.inline ?? false
```

Reply:
Yep that works. I have to say that I really like the `??` and '?.` operators!